flask_resources/deserializers/deserializers.py

```diff
@@ -14,3 +15,11 @@ class DeserializerMixin:
     def deserialize_data(self, data, *args, **kwargs):
         """Deserializes an object."""
         raise NotImplementedError()
+
+
+class NullDeserializer(DeserializerMixin):
```

Comment:
moderate: I dont manage to see in which case this would be useful... It nothing is gonna get deserialized is becuase:

1- No data is needed: the deserializer should take care of such case, or even better, not be called (which I understand is why that is there). I just think that the deserializer should not be caller or know how to handle the fact of "I want to deserialize but there is not data to deserialize").
2- There is no serializer that matches the MIMEType. For this case is just a fail (this is correct already)

Reply:
Ah I see. You mean: unless MIMEType *and* method matchup with the request's content-type, we should fail: i.e. not be lenient and not silently ignore. This should then take care of 1- as well. I will change it. 

This test is that case: https://github.com/inveniosoftware/flask-resources/pull/63/files#diff-e4c3f69dc0df8dfce5196cb1fdde465fR103 and I will make it return an error.

Reply:
Will also do removal from other PR in this one.